xtask/src/target_info.rs

```diff
@@ -63,7 +63,7 @@ fn image_info(
     command.arg(image);
     command.args(&["bash", "-c", TARGET_INFO_SCRIPT]);
     command
-        .run(msg_info, msg_info.is_verbose())
+        .run(msg_info, !msg_info.is_verbose())
```

Comment:
Why was this changed? If you want it only to use the verbosity at 2+, then that should be done, but I think it should be used if any verbosity level is set.

Reply:
I agree, not sure why I changed this